packages/stark-ui/assets/styles/_header.scss

```diff
@@ -98,7 +104,13 @@
       .stark-app-bar-content-right {
         flex-direction: column;
         width: 33%;
-        > div {
+
+		// TODO review the stark-language-selector css once the dropdown issue #640 is merged
+		.stark-language-selector {
```

Comment:
The styling should be improved because now the dropdown overlaps with the other 2 buttons in the header:

![language selector overlapping](https://user-images.githubusercontent.com/5698318/45154662-a2f7c800-b1d8-11e8-8f37-765d88c1ce21.PNG)

Reply:
Discussed with @catlabs, this will be fixed when the dropdown issue #640 is merged.